### ISSUE TITLE
ModuleNotFoundError: No module named 'sapien.physx'

### ISSUE THREAD

**Issue by eyucherin** [haosulab/ManiSkill#324]
Hello I am trying to run the test installation but I get the following error. Any insight on this would be greatly appreciated! 
I was able to run the following successfully
```
# install the package
pip install --upgrade mani_skill
# install a version of torch that is compatible with your system
pip install torch torchvision torchaudio
``` 

But when I run 
```
python -m mani_skill.examples.demo_random_action

```

I get the following error
```
(base) cherin@cherin-MS-7D99:~$ python -m mani_skill.examples.demo_random_action
Traceback (most recent call last):
  File "/usr/anaconda3/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/anaconda3/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/usr/anaconda3/lib/python3.8/site-packages/mani_skill/examples/demo_random_action.py", line 6, in <module>
    from mani_skill.envs.sapien_env import BaseEnv
  File "/usr/anaconda3/lib/python3.8/site-packages/mani_skill/envs/__init__.py", line 1, in <module>
    from .scenes import *
  File "/usr/anaconda3/lib/python3.8/site-packages/mani_skill/envs/scenes/__init__.py", line 1, in <module>
    from mani_skill.utils.registration import register_env
  File "/usr/anaconda3/lib/python3.8/site-packages/mani_skill/utils/registration.py", line 12, in <module>
    from mani_skill.vector.wrappers.gymnasium import ManiSkillVectorEnv
  File "/usr/anaconda3/lib/python3.8/site-packages/mani_skill/vector/wrappers/gymnasium.py", line 10, in <module>
    from mani_skill.utils.structs.types import Array
  File "/usr/anaconda3/lib/python3.8/site-packages/mani_skill/utils/structs/__init__.py", line 3, in <module>
    from .base import (
  File "/usr/anaconda3/lib/python3.8/site-packages/mani_skill/utils/structs/base.py", line 7, in <module>
    import sapien.physx as physx
ModuleNotFoundError: No module named 'sapien.physx'

```

Currently I have ubuntu 20.04 and the following cuda and driver versions. 

```
(base) cherin@cherin-MS-7D99:~$ nvidia-smi
Tue May  7 23:08:06 2024       
+-----------------------------------------------------------------------------------------+
| NVIDIA-SMI 550.54.15              Driver Version: 550.54.15      CUDA Version: 12.4     |
|-----------------------------------------+------------------------+----------------------+
| GPU  Name                 Persistence-M | Bus-Id          Disp.A | Volatile Uncorr. ECC |
| Fan  Temp   Perf          Pwr:Usage/Cap |           Memory-Usage | GPU-Util  Compute M. |
|                                         |                        |               MIG M. |
|=========================================+========================+======================|
|   0  NVIDIA GeForce RTX 4090        Off |   00000000:01:00.0  On |                  Off |
| 32%   34C    P8             20W /  450W |     573MiB /  24564MiB |     20%      Default |
|                                         |                        |                  N/A |
+-----------------------------------------+------------------------+----------------------+
                                                                                         
+-----------------------------------------------------------------------------------------+
| Processes:                                                                              |
|  GPU   GI   CI        PID   Type   Process name                              GPU Memory |
|        ID   ID                                                               Usage      |
|=========================================================================================|
|    0   N/A  N/A      1214      G   /usr/lib/xorg/Xorg                            305MiB |
|    0   N/A  N/A      1455      G   /usr/bin/gnome-shell                           73MiB |
|    0   N/A  N/A      2683      G   ...seed-version=20240506-050114.221000        120MiB |
|    0   N/A  N/A      7760      G   ...erProcess --variations-seed-version         46MiB |
+-----------------------------------------------------------------------------------------+
```

**Comment by StoneT2000** [comment]
Can you run pip show sapien?

this will most likely be solved by using Python >=3.9, I believe the sapien package does not support 3.8

**Comment by eyucherin** [comment]
Hi!
It seems like I am running it on python 3.8.  
I just updated to python 3.9 and it seems to work! Thank you so much :) 

```
(base) cherin@cherin-MS-7D99:~$ pip show sapien
Name: sapien
Version: 2.2.2
Summary: ['SAPIEN: A SimulAted Parted based Interactive ENvironment']
Home-page: https://sapien.ucsd.edu
Author: Sapien
Author-email: sapienaicontact@gmail.com
License: MIT
Location: /usr/anaconda3/lib/python3.8/site-packages
Requires: transforms3d, requests, opencv-python, numpy
Required-by: mani-skill2, mani-skill
```
